studio/components/interfaces/Functions/EdgeFunctionDetails.tsx

```diff
@@ -0,0 +1,298 @@
+import dayjs from 'dayjs'
+import { useRouter } from 'next/router'
+import { FC, useState, useEffect } from 'react'
+import * as Tooltip from '@radix-ui/react-tooltip'
+import { PermissionAction } from '@supabase/shared-types/out/constants'
+import { Alert, IconGlobe, IconTerminal, IconMinimize2, IconMaximize2, Button, Modal } from 'ui'
+
+import { useStore, useParams, checkPermissions } from 'hooks'
+import Panel from 'components/ui/Panel'
+import CommandRender from './CommandRender'
+import { useProjectApiQuery } from 'data/config/project-api-query'
+import { useEdgeFunctionDeleteMutation } from 'data/edge-functions/edge-functions-delete-mutation'
+
+interface Props {}
+
+// [Joshen] Next - additional configs: Verify jwt + import maps
+
+const EdgeFunctionDetails: FC<Props> = () => {
+  const router = useRouter()
+  const { functions, ui } = useStore()
+  const { ref: projectRef, id } = useParams()
+  const [showDeleteModal, setShowDeleteModal] = useState(false)
+  const [showInstructions, setShowInstructions] = useState(false)
+  const [selectedFunction, setSelectedFunction] = useState<any>(null)
+
+  const canDeleteEdgeFunction = checkPermissions(PermissionAction.FUNCTIONS_WRITE, '*')
+  const { mutateAsync: deleteEdgeFunction, isLoading: isDeleting } = useEdgeFunctionDeleteMutation()
+
+  useEffect(() => {
+    setSelectedFunction(functions.byId(id))
+  }, [functions.isLoaded, ui.selectedProject])
+
+  const { data: settings } = useProjectApiQuery({ projectRef })
+  const functionUrl = `https://${projectRef}.functions.supabase.co/${selectedFunction?.slug}`
```

Comment:
How does `functionUrl` compare to `functionsEndpoint`? And are functions always hosted on the `supabase.co` domain?

Reply:
great catch this! functionsEndpoint seems to be the more accurate one, will make the change